commit f92cec52486d748c35d8d95074cccf33187171e1 (refs/changes/69/369/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-08-29T11:58:58-07:00 (3 years, 1 month ago)
    
    DOCKER-921 Get sdc-docker-build 'make check' running on SmartOS

diff --git a/Makefile b/Makefile
index 12046ae..f47a322 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,8 @@ JSSTYLE_FILES	 = $(JS_FILES)
 JSSTYLE_FLAGS	 = -f tools/jsstyle.conf
 CLEAN_FILES += ./node_modules
 
+NODE_PREBUILT_VERSION=v0.10.46
+
 include ./tools/mk/Makefile.defs
 ifeq ($(shell uname -s),SunOS)
 	include ./tools/mk/Makefile.node_prebuilt.defs
